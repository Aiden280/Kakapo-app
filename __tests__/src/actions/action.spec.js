import {
  searchActions,
  settingActions,
  soundActions,
  themeActions,
  notifyActions
} from 'actions/';

test('[action/searchActions]', () => {
  expect(searchActions.input('test')).toMatchSnapshot();
  expect(searchActions.kakapo()).toMatchSnapshot();
  expect(searchActions.youtube('test')).toMatchSnapshot();
  expect(searchActions.soundcloud('test')).toMatchSnapshot();
  expect(searchActions.request()).toMatchSnapshot();
  expect(searchActions.requestSuccess('test', 'test')).toMatchSnapshot();
  expect(searchActions.requestError('test')).toMatchSnapshot();
});
