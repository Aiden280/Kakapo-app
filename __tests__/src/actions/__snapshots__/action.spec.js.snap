// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[action/searchActions/input] 1`] = `
Object {
  "input": "test",
  "type": "SEARCH_INPUT",
}
`;

exports[`[action/searchActions] 1`] = `
Object {
  "input": "test",
  "type": "SEARCH_INPUT",
}
`;

exports[`[action/searchActions] 2`] = `
Object {
  "type": "SEARCH_KAKAPO",
}
`;

exports[`[action/searchActions] 3`] = `
Object {
  "term": "test",
  "type": "SEARCH_YOUTUBE",
}
`;

exports[`[action/searchActions] 4`] = `
Object {
  "term": "test",
  "type": "SEARCH_SOUNDCLOUD",
}
`;

exports[`[action/searchActions] 5`] = `
Object {
  "type": "SEARCH_REQUEST",
}
`;

exports[`[action/searchActions] 6`] = `
Object {
  "items": "test",
  "service": "test",
  "type": "SEARCH_REQUEST_SUCCESS",
}
`;

exports[`[action/searchActions] 7`] = `
Object {
  "err": "test",
  "type": "SEARCH_REQUEST_ERROR",
}
`;
